src/meta/src/controller/fragment.rs

```diff
@@ -655,6 +658,33 @@ impl CatalogController {
         Ok(job_states)
     }
 
+    pub async fn list_stream_job_desc(&self) -> MetaResult<Vec<MetaTelemetryJobDesc>> {
+        let inner = self.inner.read().await;
+        let info: Vec<(TableId, Property)> = Table::find()
+            .select_only()
+            .column(table::Column::TableId)
+            .column(source::Column::WithProperties)
+            .join(JoinType::InnerJoin, table::Relation::Source.def())
+            .into_tuple()
+            .all(&inner.db)
+            .await?;
```

Comment:
```suggestion
        let info: Vec<(TableId, Option<Property>)> = Table::find()
            .select_only()
            .column(table::Column::TableId)
            .column(source::Column::WithProperties)
            .join(JoinType::LeftJoin, table::Relation::Source.def())
            .filter(table::Column::TableType.ne(TableType::Internal))
            .into_tuple()
            .all(&inner.db)
            .await?;
```
If you want to return all streaming jobs, you have to use left join and filter out internal tables. Btw, I think it's better to move this function to catalog.rs.

Reply:
fixed